front/app/components/VoteControl/VotingDisabled.tsx

```diff
@@ -74,78 +75,70 @@ const VotingDisabled = memo(({ projectId, votingDescriptor }: Props) => {
   const project = useProject({ projectId });
   const authUser = useAuthUser();
 
-  const onVerify = (event) => {
+  const onVerify = (
+    pcType: IParticipationContextType,
+    // it's theoretically possible to have a timeline project
+    // with no phases, in which case we would have no project id
```

Comment:
Maybe it's a good idea to add a test for this case?

Reply:
In practice, timeline projects without phases should never be published. So given that, I'm not going to add the test for now. (Still, I think we should automatically add one phase to a timeline project.)

I did notice that the comment is wrong though. 'project id' should be 'phase id', so I changed that.